patches/minecraft/net/minecraft/block/BlockCactus.java.patch

```diff
@@ -9,7 +9,26 @@
  {
      public static final PropertyInteger field_176587_a = PropertyInteger.func_177719_a("age", 0, 15);
      protected static final AxisAlignedBB field_185593_b = new AxisAlignedBB(0.0625D, 0.0D, 0.0625D, 0.9375D, 0.9375D, 0.9375D);
-@@ -110,8 +110,8 @@
+@@ -48,7 +48,8 @@
+             if (i < 3)
+             {
+                 int j = ((Integer)p_180650_3_.func_177229_b(field_176587_a)).intValue();
+-
```

Comment:
its about balancing the patch file size and making things look okay in the end code.
This for example is just dumb, there is no real reason to delete the whitepsace as it just ads extra onto the patch file.
And deleting the whitespace makes the code more of a clump and harder to read.